odom_topic: 'odometry'
path_topic: 'planning_node/trajectory'
cmd_topic: 'cmd_vel'
error_topic: 'errors_pub'
draw_topic: 'draw_path_pub'
global_frame: 'local_map'
base_frame: 'isns_link'
control_method: 1
